src/js/utils/template-processor.js

```diff
@@ -140,7 +144,7 @@ export function evaluateCondition(expr, state) {
     boolean: /true|false/,
     number: /-?\d+\.?\d*/,
     string: /(["'])((?:\\.|[^\\])*?)\1/,
-    operator: /[!=]=/,
+    operator: /[!=><]=|[><]/,
```

Comment:
suggestion(non-blocking): Consider deriving this regex from the set of valid `conditions` for maintainability. E.g.:

```suggestion
    operator: new RegExp(Object.keys(conditions).join('|')),
```